### PR TITLE
docs: ultra tiny typo fix

### PR DESCRIPTION
**Pull request by jensenbox**
Single character addition. 

Side Note: I have no idea what the Semantic Commit Message wants from me. I have the PR title correct. It may be broken. If it is not, it sure is not making any friends with me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This [Terraform](https://www.terraform.io/) module creates the required infrastructure needed to host [GitHub Actions](https://github.com/features/actions) self-hosted, auto-scaling runners on [AWS spot instances](https://aws.amazon.com/ec2/spot/). It provides the required logic to handle the life cycle for scaling up and down using a set of AWS Lambda functions. Runners are scaled down to zero to avoid costs when no workflows are active.
 
-> 📢 [`v3`](https://github.com/philips-labs/terraform-aws-github-runner/pull/3037) underlying loggin framework is replaces by [AWS Lambda Powertools](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/). Depending on how you handle the logging of the module a migration could be required.
+> 📢 [`v3`](https://github.com/philips-labs/terraform-aws-github-runner/pull/3037) underlying logging framework is replaces by [AWS Lambda Powertools](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/). Depending on how you handle the logging of the module a migration could be required.
```
Comment (IdanLupinskyMyPorsche) [inline README.md:7]: replaces -> replaced

Comment (idanlupinsky) [inline README.md:7]: replaces -> replaced (a tiny typo fix to add to your tiny typo fix) :)

Comment (jensenbox) [reply README.md]: done.

**Review by npalm — APPROVED** [review]
@jensenbox thanks. Even a small contribution is very welcome.

**Pull request merged**